mugoh/qstioner
# #163652235 POST a User-Created Meetup to the API Client

#### What does this PR do?
Link the meetup creation interface with its API endpoint.

#### Description of Task to be completed?
- Send a post request to create a meetup on submission of meetup details in the user interface.

#### How should this be manually tested?
Ensure the server is running. Follow the steps given [here](https://github.com/hogum/qstioner-v2) to do so.
1. Clone the repo in your machine
``` shell
$ git clone https://github.com/hogum/qstioner
```
2. Navigate to the directory `qstioner` where you have the repo cloned
``` shell
$ cd qstioner
```
3. Checkout this Feature branch
``` shell
$ git checkout ft-create-meetup-163652235
```
4. Open the `create_meetup.html` file
``` shell
$ browser-name create_meetup.html
```
or
Simply  load up on your preferred browser

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#163652235](https://www.pivotaltracker.com/story/show/163652235)
